app/components/common/core/core.js

```diff
@@ -1,4 +1,4 @@
-(function() {
+// (function() {
```

Comment:
Did you mean to comment that out? No objections either way, but figured you might want it.